ApplicationStater.py

```diff
@@ -4,9 +4,11 @@
 from Migration.ConnectionAdaptor import MysqlConnectionAdaptor
 from Migration.MetadataAdaptor import MysqlMetadataAdaptor
 from Migration.CoreAdaptor import MysqlCoreAdaptor
-
+from Migration.NosqlConnectionAdaptor import ConnectionMongodb
+from Migration.NosqlMetadaAdaptoe import MetaMongodb
```

Comment:
Change Adaptoe to Adapter. Also, rename/refactor class names. Ex: MongoDBConnection Adaptor